erpnext/stock/doctype/stock_entry/test_stock_entry.py

```
@@ -1333,7 +1333,7 @@ def test_future_negative_sle(self):
 			),
 		]
 
-		self.assertRaises(NegativeStockError, create_stock_entries, sequence_of_entries)
+		self.assertRaises(frappe.ValidationError, create_stock_entries, sequence_of_entries)
```

Ruff (0.14.14)

[error] `frappe` may be undefined, or defined from star imports

(F405)